### DISCUSSION TITLE
在Vercel部署时，Deploy中的building报错 || When deploying in Vercel, the building in Deploy reports an error

### DISCUSSION THREAD

**Discussion by TajangSec** [walinejs/waline#2274] [Q&A] [answered]
### Describe the bug | 问题描述

如图，我明明按照官方文档一步步来的
![75dd25b68d6a92c7297894da6994a74a](https://github.com/walinejs/waline/assets/63721558/3c6bd22c-9cb2-4bd2-a5b6-0a54c0bfec9b)

### Website URL | 问题网站

https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fwalinejs%2Fwaline%2Ftree%2Fmain%2Fexample

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

As shown in the picture, I clearly followed the official documentation step by step.
![75dd25b68d6a92c7297894da6994a74a](https://github.com/walinejs/waline/assets/63721558/3c6bd22c-9cb2-4bd2-a5b6-0a54c0bfec9b)

### Website URL | Problem website

https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fwalinejs%2Fwaline%2Ftree%2Fmain%2Fexample

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
same as #2264
